src/main/java/mate/academy/service/impl/MovieSessionServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.service.impl;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.dao.MovieSessionDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.MovieSession;
+import mate.academy.service.MovieSessionService;
+
+@Service
+public class MovieSessionServiceImpl implements MovieSessionService {
+
+    @Inject
+    private MovieSessionDao movieSessionDao;
+
+    @Override
+    public MovieSession add(MovieSession movieSession) {
+        return movieSessionDao.add(movieSession);
+    }
+
+    @Override
+    public MovieSession get(Long id) {
+        return movieSessionDao.get(id)
+                .orElseThrow(() -> new RuntimeException("MovieSession not found by id: " + id));
```

Comment:
The `get()` method should return an `Optional<MovieSession>` instead of throwing an exception directly. Consider returning the `Optional` and handling the exception at a higher level, if needed, to align with the task requirements.